### DISCUSSION TITLE
i am facing issue to integrate keycloak in react js  Module not found: Error: Can't resolve 'keycloak-js

### DISCUSSION THREAD

**Discussion by upendraTekdi** [keycloak/keycloak#19715] [Q&A]
package name : npm i keycloak-js

issue : Module not found: Error: Can't resolve 'keycloak-js

versions used :
  "keycloak-js": "^19.0.1",
  "react": "^17.0.0",
  "react-dom": "^17.0.0",
  "react-router-dom": "^6.4.2",
  "react-scripts": "5.0.1",

**Comment by jonkoops** [comment]
This is really not enough information to reproduce your issue, please provide a [minimal, reproducible example](https://stackoverflow.com/help/minimal-reproducible-example).
